client/tck/pom.xml

```diff
@@ -70,5 +70,9 @@
             <artifactId>assertj-core</artifactId>
             <scope>compile</scope>
         </dependency>
+        <dependency>
+            <groupId>io.smallrye</groupId>
+            <artifactId>smallrye-graphql-client</artifactId>
+        </dependency>
```

Comment:
The TCK shouldn't depend on the implementation. This will be important when we move it to the MP project.

Reply:
Ah thanks for noticing this, I added that because of a hack that I removed later anyway, so this can go away